modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -234,7 +234,9 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
     {
     public:
         int type;
-        Size kernel, stride, pad;
+        Size kernel, stride;
+        int pad_l, pad_t, pad_r, pad_b;
+        // Size pad;
```

Comment:
Let's also restore `Size pad;` but mark it using `CV_DEPRECATED`.
We need to fill values in `pad` in the same way for symmetric paddings.